nf_core/pipeline-template/{{cookiecutter.name_noslash}}/lib/Schema.groovy

```diff
@@ -45,39 +45,53 @@ class JSON {
                properties
                description
         */
-        def output_map = new LinkedHashMap()
-
-        // Lets go deeper
-        usage.each { key, val ->
-            def Map submap = usage."$key".properties // Gets the property object of the group
+        def params_map = new LinkedHashMap()
+        json_params.each { key, val ->
+            def Map group = json_params."$key".properties // Gets the property object of the group
             def sub_params = new LinkedHashMap()
-            submap.each { innerkey, value ->
-                sub_params.put("$innerkey", "$value.description")
+            group.each { innerkey, value ->
+                sub_params.put("$innerkey", [ "$value.type", "$value.description" ])
```

Comment:
What do you think about having `[string]` instead of just `string`?

```suggestion
                sub_params.put("$innerkey", [ "[$value.type]", "$value.description" ])
```

Reply:
Let me try it 👍

Reply:
Ok. Added https://github.com/nf-core/tools/pull/669/commits/95f030a0303223d95b95a1deb79afd55b7637e62